recipes/xkbcommon/all/conanfile.py

```diff
@@ -51,6 +51,8 @@ def config_options(self):
     def configure(self):
         if self.settings.os not in ["Linux", "FreeBSD"]:
             raise ConanInvalidConfiguration("This library is only compatible with Linux or FreeBSD")
+        if self.options.docs != "deprecated":
+            self.output.warn("'docs' option is deprecated. Do not use.")
```

Comment:
why not just remove an option completely?

Reply:
@SSE4 there was already quite a big discussion above.